tests/test_controllers/test_all_controllers.py

```diff
@@ -0,0 +1,93 @@
+"""
+Test all controllers on the Sawyer(Lift) task environment as a test case.
+
+The following controllers are tested:
```

Comment:
What are the test cases? Additional descriptions would be helpful!

Reply:
Added significant documentation!

Reply:
@cremebrule minor comment about some documentation - you say "e.g. the pos/ori controller should be expected to move in the x direction first, then the y direction, ..." - this description is a little confusing, what do you mean by that? it suggests that the controller is moving axis by axis sequentially

Reply:
Is this what the test is supposed to do to the arm?

Reply:
yes! exactly. I've added more documentation that is more clear and explicit